docs/installation-guide-aws.md

```diff
@@ -76,8 +76,8 @@ And any subnets for control plane components should exist inside the `vpc_cidr`
 ```bash
 export TF_VAR_subnets_bastion='["172.30.33.160/28", "172.30.33.176/28"]'
 export TF_VAR_subnets_vault='["172.30.33.48/28"]'
-export TF_VAR_subnets_elb='["172.30.33.64/28","172.30.33.80/28","172.30.33.96/28"]'
-export TF_VAR_subnets_worker='["172.30.33.112/28","172.30.33.128/28","172.30.33.144/28"]'
+export TF_VAR_subnets_elb='["172.30.33.64/27","172.30.33.96/27","172.30.33.128/27"]'
+export TF_VAR_subnets_worker='["172.30.33.192/28","172.30.33.208/28","172.30.33.224/28"]'
```

Comment:
that might come too late, as new installation platypus is already with /28
https://github.com/giantswarm/installations/blob/master/platypus/terraform/bootstrap.sh#L39-L42

Reply:
I know
I’m setting up new adidas installation with bigger subnets.
For others we will need to go through migration